app/lib/actions/katello/repository_set/disable_repository.rb

```diff
@@ -18,8 +18,8 @@ def humanized_name
           _("Disable")
         end
 
-        def plan(product, content, substitutions)
-          if repository = repository_mapper(product, content, substitutions).find_repository
+        def plan(product, content, substitutions, registry_name = nil)
```

Comment:
so here and in enable_repository, i'm curious if we should either:

1)  rename substitutions to 'yum_substitutions'
2) put both into an options hash,  :options => {:release => '6Server', :arch => 'x86_64'} or {:registry_name => '/rhel6'}